invisible_cities/io/pmaps_io_test.py

```diff
@@ -201,6 +205,18 @@ def test_load_pmaps_as_df_lazy(KrMC_pmaps_filename, KrMC_pmaps_dfs):
         assert_dataframes_equal(df_lazy, df_eager)
 
 
+def test_load_pmaps_as_df_eager_lazy(KrMC_pmaps_filename):
+    eager = pmpio.load_pmaps_as_df(KrMC_pmaps_filename, lazy=False)
+    lazy  = pmpio.load_pmaps_as_df(KrMC_pmaps_filename, lazy=True )
+    assert len(eager) == 5
+    assert all(isinstance(item, pd.DataFrame) for item in eager)
+
+    assert isinstance(lazy, Generator)
+    element = next(lazy)
+    assert len(element   ) == 5
```

Comment:
Is there a reason this spacing after `element` is the way it is? It doesn't appear to align with anything

Reply:
Probably a leftover from a previous version. Will fix.